src/app/fsat/fan-motor/fan-motor.component.ts

```diff
@@ -118,7 +118,14 @@ export class FanMotorComponent implements OnInit {
   }
 
   checkWarnings() {
-    this.warnings = this.fsatWarningService.checkMotorWarnings(this.fsat, this.settings);
+    let isMod: boolean;
```

Comment:
Don't need this, `baseline` input is a poorly named boolean that is true when it is for the baseline and false when it is a modification. just use `!this.baseline`